source/includes/_payments.md

```diff
@@ -90,7 +90,7 @@ Content-Type: application/json
 }
 ```
 
-Attribute | Description
+Arguments | Description
```

Comment:
Because an `Attribute` is not an `Argument` ^^
Attribute if for an Object and Argument for method/function, no ??